public/app/features/alerting/unified/home/Insights.tsx

```diff
@@ -173,7 +176,29 @@ export function getInsightsScenes() {
     controls: [
       new SceneReactObject({
         component: SectionSubheader,
-        props: { children: <div>Monitor the status of your system.</div> },
+        props: {
+          children: (
+            <React.Fragment>
+              <Text>
+                {' '}
```

Comment:
why do we need these spaces?

Reply:
No. Removed by https://github.com/grafana/grafana/pull/96496/commits/5234df526a18dac088d3a3c9ea5ab9fac3f85e8a